regtest.sh

```diff
@@ -343,17 +322,16 @@ default-processes() {
 # Args:
 #   spec_gen: A program to execute to generate the spec.
 #   spec_regex: A pattern selecting the subset of tests to run
-#   instances: A number of times each test case is run
 #   parallel: Whether the tests are run in parallel (T/F).  Sequential
 #     runs log to the console; parallel runs log to files.
-#   impl: one of fast_counts, python, cpp
+#   impl: one of python, or cpp
 
 _run-tests() {
   local spec_gen=$1
   local spec_regex="$2"  # grep -E format on the spec, can be empty
-  local instances=$3
-  local parallel=$4
-  local impl=$5
+  local parallel=$3
+  local impl=$4
+  local instances=1
```

Comment:
I wanted to document it, but I honestly couldn't figure out what it was for. It is particularly as it is not the last parameter so if you wish to specify the client to use this becomes a mandatory argument. Never the less I am not dismantling the ability to do this, in case we want to revive it, or change it as a one off.